qabel_provider/aws.py

```diff
@@ -0,0 +1,48 @@
+import json
+import boto3
+
+
+class Policy:
+
+    def __init__(self, user):
+        self.user = user
+        self.policy = {
+            "Version": "2012-10-17",
+            "Statement": []
+        }
+        self.append(
+            {
+                "Sid": "Stmt1444045543000",
```

Comment:
This defines the policy version and sets a statement id which is completely irrelevant in this case. The Sid is optional and can be removed, which is what I will do.